src/map/npc.c

```diff
@@ -5723,8 +5723,10 @@ static bool npc_unloadfile(const char *filepath, bool unload_mobs)
 
 	dbi_destroy(iter);
 
-	if (found) /// Refresh event cache.
+	if (found) { /// Refresh event cache.
```

Comment:
```suggestion
	if (found == true) { /// Refresh event cache.
```

Reply:
this useless for bools. it already checks for ``true``